core/src/banking_stage/consumer.rs

```diff
@@ -989,7 +992,11 @@ mod tests {
             genesis_config,
             mint_keypair,
             ..
-        } = create_slow_genesis_config(10_000);
+        } = create_genesis_config_with_leader(
+            10_000,
+            &Pubkey::new_unique(),
```

Comment:
small diff here from original - use unique instead of random...one less random in tests.